lesson_38/my-project/src/components/Cart.jsx

```diff
@@ -0,0 +1,13 @@
+import cart from './../assets/pngwing.com.png';
+import './Cart.css';
+
+const Cart = (props) => {
+  return (
+    <div className='cart'>
+        <img className='basket' src={cart} alt="img-basket" />
```

Comment:
you sometimes use quotes like `'` and sometimes `"`
choose the one style for whole project